src/answers-umd.js

```diff
@@ -428,6 +436,21 @@ class Answers {
       onFinally: config.onFinally || function () {}
     });
   }
+
+  /*
+   * Adds context as a parameter for the query API calls.
+   * @param {Object} context The context object passed in the API calls
+   */
+  setContext (context) {
+    const contextString = JSON.stringify(context);
```

Comment:
Does this blow up if context is null or undefined?

Reply:
Nope!

![image](https://user-images.githubusercontent.com/32468132/85469150-5206be80-b57b-11ea-9d0e-f97167b69ab7.png)